src/main/scala/firrtl/annotations/AnnotationUtils.scala

```diff
@@ -3,9 +3,15 @@
 package firrtl
 package annotations
 
+import net.jcazevedo.moultingyaml._
+import firrtl.annotations.AnnotationYamlProtocol._
+
 import firrtl.ir._
 
 object AnnotationUtils {
+  def toYaml(a: Annotation): String = a.toYaml.prettyPrint
+  def fromYaml(s: String): Annotation = s.parseYaml.convertTo[Annotation]
```

Comment:
Splitting it doesn't work, because we rely on the Yaml protocol to serialize a case class. I'll move `toYaml` to the class, but keep the `fromYaml` here.